spec/features/participant_update_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'updating a participant' do
+  context 'with master of ceremonies privilege' do
+    it 'updates a participant' do
+      create :participant, name: 'Foo Bar'
+      when_current_user_is :master_of_ceremonies
+      visit participants_url
+      within ('tr:nth-child(3)') do
```

Comment:
Same question - is there a more meaningful page element to attach to?

Also, these parentheses aren't needed.